src/components/AdvertsTableRow/AdvertsTableRow.tsx

```diff
@@ -91,11 +93,16 @@ const AdvertsTableRow = memo((props: TAdvertsTableRowRenderer) => {
 
     const redirectToVerification = () => {
         const searchParams = new URLSearchParams(location.search);
-        searchParams.set('poi_poa_verified', 'false');
-        history.replace({
-            pathname: location.pathname,
-            search: searchParams.toString(),
-        });
+        searchParams.set('verified', 'false');
```

Comment:
updated the param name to make it more general since now we're also checking for PNV if enabled

Reply:
just a question. why do we keep this in the URL params? I mean it can be easily manipulated by the user.
can we add a hook and check this in component level using that hook? 🤔